src/games/seabattle/Battle.tsx

```diff
@@ -89,6 +95,18 @@ export class Battle extends React.Component<IBattleProps, IBattleState> {
     }
   }
 
+  _notifyPlayer() {
+    if (this.state.playerID === this.state.currentPlayer) {
+      if (!this.state.playerNotified) {
+        this.setState({
+          ...this.state,
+          playerNotified: true,
+        });
+        return (<Notification title={'Your Turn!'} />);
```

Comment:
this will only work if the user has the tab open, which will not be the case for mobile users. We need to do server-side push notifications for mobile, but this can be a temporary solution

Reply:
That's what I thought.  I know you said this is low-priority, so we can revisit this later.